german/Korallen/Hoehle/Unterwasserstation/Person/Person.md

```diff
@@ -6,3 +6,5 @@ Er drückt Ihnen einen Stein in die Hand und sagt "Hier mein Junge, bring das in
 
 [Ihnen ist das alles nicht mehr Geheuer in dieser Hoehle, mit dem Stein in der Hand springen Sie wieder ins Wasser
  und schwimmen aus der Höhle](../../../Korallen.md)
+
+[Dann wachen Sie ploetzlich auf und bemerken, dass Sie ein Franzose/eine Französin sind.](../../../../../french/feu-de-camp.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.